### PR TITLE
added hook to check for the userId before updating

### LINKED ISSUE #12: Add hook to make sure a user can't create a Book for another user
Currently a user can add a book for another user. We need to add a hook that checks to make sure they can't do that.

### PR DESCRIPTION
**Pull request by briancbarrow**
closes #12

### CODE REVIEW

File: src/hooks/check-id.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Use this hook to manipulate incoming or outgoing data.
+// For more information on hooks see: http://docs.feathersjs.com/api/hooks.html
+const { Forbidden } = require('@feathersjs/errors')
+
+// eslint-disable-next-line no-unused-vars
+module.exports = (options = {}) => {
```
Comment (mnorth-nelnet) [inline src/hooks/check-id.js:6]: Is there a reason to disable `no-unused-vars` and have the options param? we don't seem to be using it

Comment (mikelnorth) [inline src/hooks/check-id.js:6]: Is there a reason to disable no-unused-vars and have the options param? we don't seem to be using it

**Review by mikelnorth — APPROVED** [review]

File: src/hooks/check-id.js (hunk 1 above)
Comment (briancbarrow) [reply src/hooks/check-id.js:6]: That comes in the hook when it is generated by the cli. I didn't want to get rid of things until I understand them better. I'm not sure exactly how they are used and haven't taken the time to look into it yet.

Comment (mikelnorth) [reply src/hooks/check-id.js:6]: I figured it was a long those lines, thanks

**Pull request merged**